extensions/hoarder/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hoarder",
+  "title": "Hoarder",
+  "description": "Manage hoarder bookmarks",
+  "icon": "hoarder-icon.png",
+  "author": "kdurek",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search",
+      "title": "Search Bookmarks",
+      "description": "Search Bookmarks",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "apiUrl",
+      "type": "textfield",
+      "title": "API URL",
+      "description": "The URL of the Hoarder API",
+      "required": true,
+      "placeholder": "https://hoarder.app"
```

Comment:
Can you host it yourself?

Reply:
What do you mean, is it self hostable? I provided placeholder just for the sake it should be url :D

Reply:
Oh okay! I mean it's only self-hostable for now at least, I just missed ability to find my bookmarks in raycast, prob gonna extend this extension forward